R/badge.R

```diff
@@ -134,6 +135,22 @@ use_binder_badge <- function(ref = git_default_branch(), urlpath = NULL) {
   invisible(TRUE)
 }
 
+#' @rdname badges
+#' @param cloudurl A link to an existing [RStudio Cloud](https://rstudio.cloud) project
```

Comment:
Do we need to say anything more about how to get this URL? Or link to any Cloud docs? Pardon my ignorance, I just haven't ever sent a bunch of people to an RStudio Cloud project like this.